asf_floorplan_interpreter/pipeline/predict_floorplan.py

```diff
@@ -179,7 +187,7 @@ def predict_labels(self, image_url, correct_kitchen=True, conf_threshold=0):
         # We found the results are best if the output says at least one kitchen per floorplan
         if correct_kitchen:
             if "KITCHEN" in label_counts:
-                label_counts["KITCHEN"] = max(label_counts["KITCHEN"], 1)
+                label_counts["KITCHEN"] = 1
```

Comment:
@sqr00t I think this might be the only change you should be aware of

Reply:
would be fine if we expect there not to be 2 kitchens. might later update the docs to say there is at least and only one kitchen